config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "callbacks" }
```

Comment:
Please use ruby new syntax styling
```suggestion
  devise_for :users, controllers: { omniauth_callbacks: 'callbacks' }

```